src/action.c

```diff
@@ -144,11 +169,31 @@ action_str_from_arg(struct action_arg *arg)
 	return ((struct action_arg_str *)arg)->value;
 }
 
+static char *
+str_from_key(struct action *action, const char *key, char *default_value)
```

Comment:
`const char *` for both the argument and return type?

Reply:
Have added const.